weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/OverflowMenu.tsx

```diff
@@ -45,7 +47,12 @@ export const OverflowMenu: FC<{
     <>
       {confirmDelete && (
         <ConfirmDeleteModal
-          calls={selectedCalls}
+          entity={entity}
```

Comment:
While in practice, all the calls for a page are the same entity/project, it seems weird to pass this in from the caller. Instead, i would extract the first entity/project from the incoming calls inside the delete modal. As a bonus, since the delete calls requires an entity, project, you could extract unique entity/project pairs, then delete those calls in groups.

Reply:
right, yeah ill add that group delete.